tests/examples/logging/playground/printstuff/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec multilog t s10485760 n5 .
```

Comment:
Same: If we require each service to do this right, it will mostly be done wrong.
Need to see if we can make this behavior part of the framework.